prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -2,27 +2,55 @@
 
 layout: page_v2
 title: Custom or No mediation
-description: Integration of Prebid SDK withou primaty Ad Server
+description: How to integrate the Prebid SDK without a primary Ad Server
 sidebarType: 2
 
 ---
 
+
+
 # Custom Bidding Integration
 {:.no_toc}
 
-You can use Prebid SDK to monetize your app with a custom ad server or even without it. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without primary ad server and its SDK.
+## Before you start
```

Comment:
I think this section should be a short "intro" to what's going to happen.

1. You're going to initialize the SDK
2. You're going to set up AdUnits of various types
3. You're going to initialize the auction
4. Finally, you're going to render the results.

Then add a new section for "Initialization"

Reply:
Definitely progress here, but I still question the "Transport API" thing. IMO, the sections should be: Initialize the SDK, Set Up Adunits, Initialize the Auction, and Render Results.